### DISCUSSION TITLE
Best way to make stickers

### DISCUSSION THREAD

**Discussion by edsousa84** [leozide/leocad#854] [Q&A]
Hello can some one tell me what would be the best way to add stickers to a model I have made with leocad?
I have a image in png that I can remake on a SVG format if needed.

I can see two options but I dont know how to satrt on any of them:
1) create a parte with the sticker already applyed
2) create a part that is the sticker and add it to the part were the sticker goes.

(I would perfer the second one)

**Comment by rsbx** [comment]
The modern method used to create decorated LDraw parts is documented on 
the LDraw.org site, under the Documentation tab, TEXMAPs (uses PNG 
files). The older method is to design the decoration into the part 
itself. Both require becoming familiar with how LDraw parts are defined; 
also documented on the LDraw site.

**Reply by edsousa84** [reply]
is leocad compatible with both methonds? Modern and Old?

**Comment by rsbx** [comment]
Yes, as long you create valid LDraw parts.

**Reply by edsousa84** [reply]
Thank you.

I think I manage to make a sticker as a example but Iam having trouble on how I cad add it as a part?
Where do I have to put the .dat file?

I tryed on the ldraw folder parts but it does nos show on leocad.
